src/app/components/shell/footer/footer.hbs

```diff
@@ -1,64 +1,62 @@
 <style scoped>@import url(/app/components/shell/footer/footer.css);</style>
 <footer class="footer">
+    <div class="hero-quote">Access. The Future of Education.</div>
+
     <div class="copyright">
-        <nav>
-            <ul>
-                <li><a href="/license">Licensing</a></li>
-                <li><a href="/tos">Terms of Use</a></li>
-                <li><a href="/contact">Contact</a></li>
-            </ul>
-        </nav>
+        <ul>
+            <li>
+                <a href="/licensing">Licensing</a>
```

Comment:
This is weird. A list with one item that contains three distinct things? Is there a reason not to make this just a div with a class?